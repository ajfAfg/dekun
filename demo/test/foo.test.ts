import { extract } from "@ajfafg/dekun";

describe("Demo", () => {
  it("sum", () => {
    expect(extract("src/foo.ts", "sum")).toMatchSnapshot();
  });
  it("prod", () => {
    expect(extract("src/foo.ts", "prod")).toMatchSnapshot();
  });
});
