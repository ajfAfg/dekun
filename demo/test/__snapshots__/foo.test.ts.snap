// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Demo prod 1`] = `
"function prod(xs: [
    number
]): number {
    return xs.reduce((x, y) => x * y, 1);
}"
`;

exports[`Demo sum 1`] = `
"export const sum = (xs: [
    number
]): number => xs.reduce((x, y) => x + y, 0);"
`;
